source/controllers/game.py

```diff
@@ -30,13 +30,16 @@ def join_game_controller(args, sock):
     game_id = args['game_id']
     player_name = str(args['player_name']).strip()
     password = str(args['password']).strip()
-
-    try:
-        join_game(player_name, game_id, password, sock)
-    except TakiException as e:
-        return e.response()
-    except Exception:
-        return Responses.INTERNAL_ERROR
+    if does_player_exiset(sock):
```

Comment:
Shouldn't the game object check if the socket already exists? (in the "add_player" func in game.py)